tests/forward_diffusion_test.py

```diff
@@ -0,0 +1,72 @@
+import torch
+import seaborn as sns
+import matplotlib.pyplot as plt
+from torch.utils.data import DataLoader
+
+from ecg_segmentation_dataset import ECGDataset
+from models.forward_diffusion import ForwardDiffusion
+
+
+def preprocess_dataset(dataset_name: str, batch_size: int, num_workers: int):
+    train_ds = ECGDataset(dataset_name, split="train")
+    val_ds = ECGDataset(dataset_name, split="validation")
+    test_ds = ECGDataset(dataset_name, split="test")
+
+    # dataloaders
+    train_dataloader = DataLoader(train_ds, batch_size=batch_size, num_workers=num_workers)
+    val_dataloader = DataLoader(val_ds, batch_size=batch_size, num_workers=num_workers)
+    test_dataloader = DataLoader(test_ds, batch_size=batch_size, num_workers=num_workers)
+
+    return train_dataloader, val_dataloader, test_dataloader
+
+
+if __name__ == "__main__":
+    # load data
+    _, _, test_dataloader = preprocess_dataset("roszcz/ecg-segmentation-ltafdb", 4, 1)
```

Comment:
```suggestion
    _, _, test_dataloader = preprocess_dataset(
      dataset_name="roszcz/ecg-segmentation-ltafdb",
      batch_size=4,
      num_workers=1,
    )
```

It's much easier to understand with named arguments